### ISSUE TITLE
Fix Vertice calculation upon exporting ped (head) ydd

### ISSUE THREAD

**Issue by ook3D** [Sollumz/Sollumz#646]
![image](https://github.com/Skylumz/Sollumz/assets/47336113/b05dad08-cbd2-4e8e-8e31-596438a2f04b)

appears that head models need to be very specific regarding their vertice count, and IIRC everything gets automatically triangulated apon export, maybe an option to disable this when exporting could fix this.

**Comment by Disquse** [comment]
vertice count *and* order

**Comment by coltfox** [comment]
>and IIRC everything gets automatically triangulated apon export, maybe an option to disable this when exporting could fix this

That has nothing to do with this. You have to triangulate the meshes on export, as the game engine (and any rendering engine for that matter) renders in triangles. N-gons is just a feature of Blender. Even Blender actually renders the meshes as triangles.

**Comment by ook3D** [comment]
> That has nothing to do with this. You have to triangulate the meshes on export, as the game engine (and any rendering engine for that matter) renders in triangles. N-gons is just a feature of Blender. Even Blender actually renders the meshes faces as triangles.

ah, i forgot about that

**Comment by MaaximGL** [comment]
@ook3D  Yes, you're right. If the vertex order is maintained, the head works fine. I tested this by moving vertices in the exported head through Sollumz, roughly comparing them with vanilla values. With the correct order, you can even change the shader for individual parts that can be blended.
